.github/workflows/major-release-tag.yml

```diff
@@ -0,0 +1,21 @@
+name: Move Major Release Tag
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  movetag:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Get major version num and update tag
+      run: |
+        VERSION=${GITHUB_REF#refs/tags/}
+        MAJOR=${VERSION%%.*}
+        git config --global user.name 'OctoMind'
```

Comment:
not sure about this part, please review @Germandrummer92

Reply:
see https://github.com/richardsimko/update-tag maybe?